src/Helpers/CadastralNumberInfo.php

```diff
@@ -9,51 +9,54 @@ class CadastralNumberInfo
     /**
      * Код субъекта.
      *
-     * @var string
+     * @var int
      */
-    protected $region_code;
+    protected $district_code;
 
     /**
      * Номер района.
      *
-     * @var string
+     * @var int
      */
-    protected $district_code;
+    protected $area_code;
 
     /**
      * Номер квартала.
      *
      * @var string
      */
-    protected $quarter_code;
+    protected $section_code;
 
     /**
      * Номер участка.
      *
      * @var string
      */
-    protected $area_code;
+    protected $parcel_number;
 
     /**
      * CadastralNumberInfo constructor.
      *
```

Comment:
Please add `@limk <...>` this link to the relevant naming source

Reply:
Move `@see` block and put over `constructor`?